packages/sdk/chainDeploy/mainnets/ethereum.ts

```diff
@@ -0,0 +1,130 @@
+import { ethereum } from "@midas-capital/chains";
+import { assetSymbols, underlying } from "@midas-capital/types";
+import { ethers } from "ethers";
+
+import { ChainDeployConfig, deployChainlinkOracle, deployErc4626PriceOracle } from "../helpers";
+import { ChainDeployFnParams, ChainlinkAsset, ChainlinkFeedBaseCurrency, ERC4626Asset } from "../helpers/types";
+import {} from "../helpers/oracles/erc4626";
+
+const assets = ethereum.assets;
+const USDC = underlying(assets, assetSymbols.USDC);
+const WETH = underlying(assets, assetSymbols.WETH);
+
+export const deployConfig: ChainDeployConfig = {
+  wtoken: WETH,
+  nativeTokenName: "Wrapped ETH",
+  nativeTokenSymbol: "ETH",
+  stableToken: USDC,
+  nativeTokenUsdChainlinkFeed: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419",
+  blocksPerYear: ethereum.specificParams.blocksPerYear.toNumber(), // 12 second blocks, 5 blocks per minute// 12 second blocks, 5 blocks per minute
+  uniswap: {
+    hardcoded: [],
+    uniswapData: [],
+    pairInitHashCode: ethers.utils.hexlify("0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f"),
+    uniswapV2RouterAddress: "0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45",
+    uniswapV2FactoryAddress: "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f",
+    uniswapV3FactoryAddress: "0x1F98431c8aD98523631AE4a59f267346ea31F984",
+    uniswapOracleInitialDeployTokens: [],
+    uniswapOracleLpTokens: [],
+    flashSwapFee: 25,
```

Comment:
looks like the fee is 30 https://docs.uniswap.org/contracts/v2/guides/smart-contract-integration/using-flash-swaps

Reply:
👍